src/models/ServerStyleSheet.js

```diff
@@ -24,7 +24,7 @@ class ServerTag implements Tag {
   }
 
   addComponent(componentId: string) {
-    if (this.components[componentId]) {
+    if (process.env.NODE_ENV !== 'production' && this.components[componentId]) {
       throw new Error(`Trying to add Component '${componentId}' twice!`)
```

Comment:
I'd rather like us to use `invariant` and use the babel plugin to transpile away the error message in prod. Makes sense? :smile:

Otherwise removing the errors would pose a little bit of confusion if someone is currently relying on these safeguards in prod (even if no one should)

Reply:
> I'd rather like us to use invariant and use the babel plugin to transpile away the error message in prod.

Really? I hate `invariant`, I still don't understand how it works 🙄 So unintuitive. Imo this is pretty clear and if it works, it works?

> Otherwise removing the errors would pose a little bit of confusion if someone is currently relying on these safeguards in prod

We never make any guarantees about those, so I'm don't think that's an argument for not removing them. 99.99999% of our users will benefits, the two that check for this in their tests aren't very smart :wink: Same with #1362, somebody could be using the `lib/` dir directly. Do we care about them? Not really.